src/pages/designing/get-started.mdx

```diff
@@ -13,85 +13,98 @@ running.
+## Step 2: Access the tools
 
-Carbon expresses the IBM Design Language in product and delivers it through
-tools for designers and developers including guidance, tooling, components, and
-support. Learn what drives IBM’s design philosophy and principles so that you
-can make informed decisions in your product work.
+_For IBMers only:_
 
-The IBM brand systems have been developed for various IBM businesses, audiences,
-categories, and offerings. Read them to understand the rationale behind every
-visual and verbal detail to ensure your work is built on these important
-foundations. _This content is accessible to IBMers only._
+IBMers should get a license for Figma, our primary design kit tool. You may also
+access tools we no longer prioritize, Sketch, Adobe XD, and Axure, by heading to
+the [Design Toolbox](https://w3.ibm.com/design/toolbox/).
 
 <Row className="resource-card-group">
-<Column colLg={4} colMd={4} noGutterSm>
+<Column colMd={4} colLg={4} noGutterSm>
   <ResourceCard
-    subTitle="IBM Design Language"
-    href="https://www.ibm.com/design/language/"
-    >
+    subTitle="Get Figma"
+    href="https://w3.ibm.com/design/toolbox/?&_ga=2.49792614.1500760159.1682969477-2020417520.1680805833#/ui-design-tools/figma/README"
+    actionIcon="launch">
 
-![Bee icon](/images/bee.svg)
+![Figma icon](/images/figma.svg)
 
   </ResourceCard>
+  </Column>
+</Row>
 
-</Column>
-<Column colLg={4} colMd={4} noGutterSm>
-  <ResourceCard
-    subTitle="IBM Brand Systems"
-    href="https://www.ibm.com/brand/systems"
-    >
-
-![Bee icon](images/bee.svg)
+## Step 3: Get the design kits
+
+We maintain the core Carbon kits in [Figma](/designing/kits/figma). These kits
+include all the Carbon core components and styles for reuse in your product and
+web experiences. By using the kits you will automatically receive updates made
+to the Carbon libraries, ensuring your designs stay up-to-date with the latest
+releases. And when handing off to your developers there won't be confusion about
+components and their behaviors.
+
+Head on over to the [Design kits](/designing/kits/figma) section and follow the
+instructions to get set up with Figma.
+
+### Core design kits
+
+Everyone using Carbon should use core design kits listed below as a starting
+point for building user interfaces. This brings consistency and speed, allowing
+teams to focus on solving more specific user tasks. Core design kits include
+elements, core components, wireframes, and guidelines.
+
+| Maintainer    | Design kit                                                                                                                           | Type         |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------ | ------------ |
+| IBM Brand     | [Figma IBM pictograms](https://www.figma.com/file/PkUl9UBuvA41GPpyl84NBc/Pictograms---IBM-Design-Language)                           | `Elements`   |
```

Comment:
```suggestion
| IBM Design Language | [Figma IBM pictograms](https://www.figma.com/file/PkUl9UBuvA41GPpyl84NBc/Pictograms---IBM-Design-Language)                           | `Elements`   |

I think these are more considered IBM Design Language rather than IBM Brand.
```